java/configuration/src/main/java/sleeper/configuration/properties/UserDefinedInstanceProperty.java

```diff
@@ -538,279 +540,286 @@ public interface UserDefinedInstanceProperty extends InstanceProperty {
             .defaultValue("7")
             .validationPredicate(Utils::isPositiveInteger)
-            .build();
+            .propertyGroup(PropertyGroup.QUERY).build();
     UserDefinedInstanceProperty DEFAULT_RESULTS_ROW_GROUP_SIZE = named("sleeper.default.query.results.rowgroup.size")
             .description("The default value of the rowgroup size used when the results of queries are written to Parquet files. The\n" +
                     "value given below is 8MiB. This value can be overridden using the query config.")
             .defaultValue("" + (8 * 1024 * 1024))
-            .build(); // 8 MiB
+            // 8 MiB
+            .propertyGroup(PropertyGroup.QUERY).build();
     UserDefinedInstanceProperty DEFAULT_RESULTS_PAGE_SIZE = named("sleeper.default.query.results.page.size")
             .description("The default value of the page size used when the results of queries are written to Parquet files. The\n" +
                     "value given below is 128KiB. This value can be overridden using the query config.")
             .defaultValue("" + (128 * 1024))
-            .build(); // 128 KiB
+            // 128 KiB
+            .propertyGroup(PropertyGroup.QUERY).build();
 
     // Dashboard
     UserDefinedInstanceProperty DASHBOARD_TIME_WINDOW_MINUTES = named("sleeper.dashboard.time.window.minutes")
             .description("The period in minutes used in the dashboard.")
             .defaultValue("5")
             .validationPredicate(Utils::isPositiveInteger)
-            .build();
+            .propertyGroup(PropertyGroup.COMMON).build();
 
     // Logging levels
     UserDefinedInstanceProperty LOGGING_LEVEL = named("sleeper.logging.level")
             .description("The loging level for logging Sleeper classes. This does not apply to the MetricsLogger which is always set to INFO.")
-            .build();
+            .propertyGroup(PropertyGroup.LOGGING).build();
     UserDefinedInstanceProperty APACHE_LOGGING_LEVEL = named("sleeper.logging.apache.level")
             .description("The loging level for Apache logs that are not Parquet.")
-            .build();
+            .propertyGroup(PropertyGroup.LOGGING).build();
     UserDefinedInstanceProperty PARQUET_LOGGING_LEVEL = named("sleeper.logging.parquet.level")
             .description("The loging level for Parquet logs.")
-            .build();
+            .propertyGroup(PropertyGroup.LOGGING).build();
     UserDefinedInstanceProperty AWS_LOGGING_LEVEL = named("sleeper.logging.aws.level")
             .description("The loging level for AWS logs.")
-            .build();
+            .propertyGroup(PropertyGroup.LOGGING).build();
     UserDefinedInstanceProperty ROOT_LOGGING_LEVEL = named("sleeper.logging.root.level")
             .description("The loging level for everything else.")
-            .build();
+            .propertyGroup(PropertyGroup.LOGGING).build();
```

Comment:
The logging level properties all have a spelling mistake in the description, "loging" vs "logging".